package.json

```diff
@@ -8,13 +8,11 @@
   "license": "MIT",
   "scripts": {
     "start:dev": "nodemon ./bin/www",
-    "seed":
-    "sequelize db:seed:undo:all && sequelize db:seed --seed levels && sequelize db:seed --seed categories && sequelize db:seed --seed status && sequelize db:seed --seed locations && sequelize db:seed --seed roles && sequelize db:seed --seed incidents && sequelize db:seed --seed users && sequelize db:seed --seed reporters && sequelize db:seed --seed witnesses",
+    "seed": "sequelize db:seed:undo:all && sequelize db:seed --seed levels && sequelize db:seed --seed categories && sequelize db:seed --seed status && sequelize db:seed --seed locations && sequelize db:seed --seed roles && sequelize db:seed --seed incidents && sequelize db:seed --seed users && sequelize db:seed --seed reporters && sequelize db:seed --seed witnesses",
```

Comment:
If we are running all seeds, why can't we just use `sequelize db:seed:all`

Reply:
Reason being: `sequelize db:seed:all` seeds in alphabetical order.That causes errors because the relationships of the seeders doesn't work like that. i.e `incidents belong to locations. Incidents cannot seed before locations.` The other option is to rename the files in a certain alphabetical/numerical order so it seeds in order of relationship.